.gitignore

```diff
@@ -10,8 +10,8 @@ env
 # tmp folder
 tmp/
 
-# Listenbrainz listens dump
-listenbrainz-listens*
+# Listenbrainz dumps
+listenbrainz-*dump*.tar.*
```

Comment:
This one doesn't match the spark dumps which are in tar format only without gzip/bzzip2 compression, eg: `listenbrainz-spark-dump-853-20220415-040003-full.tar`. probably remove the `.` after tar and it should work fine in all cases then.